src/web/app/components/sortable-table/sortable-table.module.ts

```diff
@@ -11,6 +18,8 @@ import { SortableTableComponent } from './sortable-table.component';
   imports: [
     CommonModule,
     NgbModule,
+    ContributionQuestionStatisticsModule,
+    DynamicModule.withComponents([ContributionComponent, ContributionRatingsListComponent]),
```

Comment:
One annoying thing is that the caller must remember to register their dynamic component here. Maybe should document this somewhere? 

From the docs [here](https://github.com/gund/ng-dynamic-component#readme), it seems like this step can be skipped if we upgrade to Angular 9, but since we are at Angular 8, we are using [this doc](https://github.com/gund/ng-dynamic-component/blob/56d3d9f49da88df944d87e9154ef5c0e283c10e5/README.md) instead, which requires the registration of these components.

Reply:
> Will open issue to track this as a reminder if we ever upgrade to Angular 9.

We will do soon.

Reply:
Done https://github.com/TEAMMATES/teammates/pull/10247/commits/d4934e3ffc224d8fc1876fd630c9972b5b364716